docs/bbi/nonmem/run/local/local.md

```diff
@@ -9,30 +9,16 @@ Specifies to run the targeted model on the local machine
 ### Options
 
 ```
-      --cacheDir string       directory path for cache of nonmem executables for NM7.4+
-      --cacheExe string       name of executable stored in cache
```

Comment:
Do all of these options no longer exist?

Reply:
They're not implemented anymore. They were from the original incarnation of Babylon, but are not actually implemented anywhere now.